webscraping.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import pandas as pd
+from bs4 import BeautifulSoup
+
+header = {'User-Agent': "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) "
+                        "Chrome/89.0.4389.128 Safari/537.36",
+          "X-Amzn-Trace-Id": "Root=1-608078e5-5ab47ee82924df8667c3f0e7"}
+base_url = "https://www.amazon.ca/s?k="
+
+
+# Function for searching amazon products given product name
+def get_amazon_search(search_products):
+    url = "https://www.amazon.ca/s?k=" + search_products
+    print(url)
+    page = requests.get(url, cookies={}, headers=header)
+    if page.status_code == 200:
+        return page
+    else:
+        return "Error"
+
+
+# Function for searching amazon products given product ID
+def search_product_id(asin):
+    url = "https://www.amazon.ca/dp/" + asin
+    print(url)
+    page = requests.get(url, cookies={}, headers=header)
+    if page.status_code == 200:
+        return page
+    else:
+        return "Error"
+
+
+# Function for scraping amazon product reviews
+def scrape_reviews(review_link):
+    url = "https://www.amazon.ca" + review_link
+    print(url)
+    page = requests.get(url, cookies={}, headers=header)
+    if page.status_code == 200:
+        return page
+    else:
+        return "Error"
+
+
+# Function for scraping amazon product names
+def get_product_names(product):
+    product_names = []
+    response = get_amazon_search(product)
+    soup = BeautifulSoup(response.content)
+    for i in soup.findAll("span", {'class': 'a-size-base-plus a-color-base a-text-normal'}):  # the tag which is
```

Comment:
Whenever you hardcode something into your code, you expose yourself to potential bugs. 
Transfer all constants to a config file and read from that file wherever necessary. In this line, you can move the strings over to the config file.